From bd83f790224bcff6fc795f6fd1e203795ef72df0 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Mon, 8 Apr 2013 15:08:05 -0700
Subject: remove linkage to $OUT/bootloader

Bootloader partitions are now dynamically created at runtime,
we don't need this or the ifeq guard.

Change-Id: Ib086c9f965d176fd1eab40e334f63044bf416e07
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 Android.mk |   31 -------------------------------
 1 files changed, 0 insertions(+), 31 deletions(-)

diff --git a/Android.mk b/Android.mk
index b70aa0d..359b807 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,36 +1,5 @@
-ifneq ($(strip $(TARGET_NO_BOOTLOADER)),true)
-ifeq ($(TARGET_USE_SYSLINUX),true)
-
 include $(all-subdir-makefiles)
 
 SYSLINUX_BIN := $(HOST_OUT_EXECUTABLES)/syslinux
 SYSLINUX_BASE := $(HOST_OUT)/usr/lib/syslinux
 
-SYSLINUX_MK_IMG := external/syslinux/utils/android-image.sh
-
-# TARGET_SYSLINUX_FILES - any splash screens, com32 modules
-# TARGET_SYSLINUX_CONFIG - file to use as syslinux.cfg
-# These should be defined in BoardConfig.mk per-product
-
-ifdef TARGET_SYSLINUX_IMAGE_EXTRA_SPACE
-	SYSLINUX_EXTRA_SPACE_PARAM := --extra-size $(TARGET_SYSLINUX_IMAGE_EXTRA_SPACE)
-endif
-
-$(PRODUCT_OUT)/bootloader: \
-		$(TARGET_SYSLINUX_FILES) \
-		$(TARGET_SYSLINUX_CONFIG) \
-		$(SYSLINUX_BIN) \
-		$(SYSLINUX_MK_IMG)
-	$(call pretty, "Target SYSLINUX image: $@")
-	$(SYSLINUX_MK_IMG) $(SYSLINUX_EXTRA_SPACE_PARAM) \
-		--syslinux $(SYSLINUX_BIN) \
-		--tmpdir $(call intermediates-dir-for,EXECUTABLES,syslinux-img)/syslinux-img \
-		--config $(TARGET_SYSLINUX_CONFIG) \
-		--output $@ \
-		$(TARGET_SYSLINUX_FILES)
-
-.PHONY: syslinux-image
-syslinux-image: $(INSTALLED_BOOTLOADER_MODULE)
-
-endif
-endif
-- 
1.7.4.1

